dashboard/app/views/levels/_multi_answer.html.haml

```diff
@@ -1,6 +1,10 @@
 / Show the correct answer to authorized teachers
+- multi_answer_characters = ("A".."Z").to_a
 - if current_user.try(:authorized_teacher?) && !@script.try(:pd?)
-  - correct_answers = data['answers'].select { |answer| answer['correct'] } if data['answers']
+  - correct_answers = data['answers'].each_with_index.map do |answer, index|
+    - next unless answer['correct']
+    - {text: answer['text'], index: index}
```

Comment:
possibly you could just calculate the answer-letter here instead of below so it's more obvious where it comes from? You can do some ascii-arithmetic stuff here instead of looking up an array too, eg:
```
irb> index = 2
irb> ("A".ord + index).chr
=> "C"
```

Or it might be worth defining a method to do this in the model similar to how you did it with `correct_answer_indexes`...